megaqc/api/utils.py

```diff
@@ -183,9 +175,9 @@ def handle_report_data(user, report_data):
             existing_plot_config = db.session.query(PlotConfig).filter(PlotConfig.config_type==report_data['report_plot_data'][plot]['plot_type'], PlotConfig.config_name==plot, PlotConfig.config_dataset==dataset_name).first()
             if not existing_plot_config:
                 new_plotcfg_cnt += 1
-                config_id = PlotConfig.get_next_id()
+#                 config_id = PlotConfig.get_next_id()
```

Comment:
Can we just delete these commented lines?

Reply:
Yeah, that was just a lazy way to see where `X_id` was being used so I could remove it. I'll be doing some cleanup after I finish testing.

Reply:
Yeah, I figured that was the case as you did say that it was a first pass.. 👍